packages/dapp/src/providers/I18nProvider.tsx

```diff
@@ -42,6 +45,12 @@ export const I18NProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
     });
 
     if (!languageMode) {
+      trackAttribute({
+        data: {
+          [TrackingUserProperties.Language]: i18n.language,
+          [TrackingUserProperties.DefaultLanguage]: i18n.language,
+        },
+      });
```

Comment:
What do we want to track here - language change or user language on page load? This should be tested carefully.

Reply:
My idea was: In order to track languages, it´d be interesting to know:
- default language on pageload
- active language after it´s been changed

So, if there is no language set in localStorage, it should be hitting that function and save both values as user_properties.
The default language shall not change ever, but active language will be modified once another language has been chosen by the user (in the menu).

Yes, this needs to be tested.

Reply:
I am also updating the Notion documentation to keep everything structured and list events / user_properties / custom dimensions